src/components/Company/CompanyName/CorrectCompanyName.vue

```diff
@@ -59,16 +59,18 @@ export default class CorrectCompanyName extends Vue {
   /** Watch for form submission and emit results. */
   @Watch('formType')
   private async onSubmit (): Promise<any> {
+    console.log(this.formType)
```

Comment:
Please remove this in a future commit.
Also the console.log on line 64.

Reply:
Shooooot...

Reply:
Will do